public/app/features/explore/Logs.tsx

```diff
@@ -80,8 +80,8 @@ interface State {
 }
 
 export class UnthemedLogs extends PureComponent<Props, State> {
-  flipOrderTimer: NodeJS.Timeout;
-  cancelFlippingTimer: NodeJS.Timeout;
+  flipOrderTimer?: NodeJS.Timeout;
+  cancelFlippingTimer?: NodeJS.Timeout;
```

Comment:
you can get rid of NodeJS annotations and replace them with number if wou change setTimeout/clearTimeout to window.setTimeout/window.clearTimeout